test/api/utils.js

```diff
@@ -12,109 +12,14 @@
-		LSK = {
-			port: defaultPort,
-			sendRequest: sandbox.stub().resolves(sendRequestResult),
-		};
-	});
-
-	describe('#netHashOptions', () => {
-		const { netHashOptions } = utils;
-		let result;
-
-		beforeEach(() => {
-			result = netHashOptions({ port: defaultPort });
-		});
-
-		it('should return an object with a testnet nethash', () => {
-			const { testnet } = result;
-			testnet.should.have.property('Content-Type').and.be.type('string');
-			testnet.should.have.property('nethash').and.be.type('string');
-			testnet.should.have.property('broadhash').and.be.type('string');
-			testnet.should.have.property('os').and.be.type('string');
-			testnet.should.have.property('version').and.be.type('string');
-			testnet.should.have.property('minVersion').and.be.type('string');
-			return testnet.should.have.property('port').and.be.type('number');
-		});
-		it('should return an object with a mainnet nethash', () => {
-			const { mainnet } = result;
-			mainnet.should.have.property('Content-Type').and.be.type('string');
-			mainnet.should.have.property('nethash').and.be.type('string');
-			mainnet.should.have.property('broadhash').and.be.type('string');
-			mainnet.should.have.property('os').and.be.type('string');
-			mainnet.should.have.property('version').and.be.type('string');
-			mainnet.should.have.property('minVersion').and.be.type('string');
-			return mainnet.should.have.property('port').and.be.type('number');
-		});
-	});
-
-	describe('#getURLPrefix', () => {
-		const { getURLPrefix } = utils;
-
-		it('should return http when ssl is set to false', () => {
-			const ssl = false;
-			const result = getURLPrefix({ ssl });
-			return result.should.be.equal('http');
-		});
-
-		it('should return https when ssl is set to true', () => {
-			const ssl = true;
-			const result = getURLPrefix({ ssl });
-			return result.should.be.equal('https');
-		});
-	});
-
-	describe('#getFullURL', () => {
-		const { getFullURL } = utils;
-		const URLPrefix = 'ftp';
-
-		let getURLPrefixStub;
-		let restoreGetURLPrefixStub;
-		let result;
-
-		beforeEach(() => {
-			getURLPrefixStub = sandbox.stub().returns(URLPrefix);
-			// eslint-disable-next-line no-underscore-dangle
-			restoreGetURLPrefixStub = utils.__set__('getURLPrefix', getURLPrefixStub);
-			result = getFullURL(LSK);
-		});
-
-		afterEach(() => {
-			restoreGetURLPrefixStub();
-		});
-
-		it('should get the URL prefix', () => {
-			const { ssl } = LSK;
-			return getURLPrefixStub.should.be.calledWithExactly({ ssl });
-		});
-
-		it('should add the prefix to the node URL and the port', () => {
-			return result.should.equal(`${URLPrefix}://${LSK.node}:${defaultPort}`);
-		});
-
-		it('should not include a port if not set', () => {
-			delete LSK.port;
-			result = getFullURL(LSK);
-			return result.should.equal(`${URLPrefix}://${LSK.node}`);
-		});
-	});
+const defaultBaseURL = 'http://localhost:8080';
+const path = '/resources';
+const defaultURL = `${defaultBaseURL}/api${path}`;
```

Comment:
Prefer nesting these under the suite.

Reply:
`const defaultURL = `${defaultBaseURL}/api${path}`;` is not exactly related to the test here.
What is the point of defining it separately and putting it under suite?